cypress/e2e/circulation-log/marked-as-missing.cy.js

```diff
@@ -0,0 +1,87 @@
+import moment from 'moment';
+import TopMenu from '../../support/fragments/topMenu';
+import SearchPane from '../../support/fragments/circulation-log/searchPane';
+import getRandomPostfix from '../../support/utils/stringTools';
+import permissions from '../../support/dictionary/permissions';
+import UsersSearchPane from '../../support/fragments/users/usersSearchPane';
+import UsersCard from '../../support/fragments/users/usersCard';
+import devTeams from '../../support/dictionary/devTeams';
+import testTypes from '../../support/dictionary/testTypes';
+import Users from '../../support/fragments/users/users';
+import UserEdit from '../../support/fragments/users/userEdit';
+import ServicePoints from '../../support/fragments/settings/tenant/servicePoints/servicePoints';
+import Checkout from '../../support/fragments/checkout/checkout';
+import LoansPage from '../../support/fragments/loans/loansPage';
+import InventoryInstances from '../../support/fragments/inventory/inventoryInstances';
+import UserLoans from '../../support/fragments/users/loans/userLoans';
+
+let user;
+let servicePointId;
+const item = {
+  name: `ItemName${getRandomPostfix()}`,
+  barcode: `123${getRandomPostfix()}`,
+};
+
+describe('circulation log', () => {
+  before('creating user and checking out item', () => {
+    cy.createTempUser([
+      permissions.inventoryAll.gui,
+      permissions.circulationLogAll.gui,
+    ])
+      .then(userProperties => {
+        user = { ...userProperties };
+        cy.login(user.username, user.password, { path: TopMenu.circulationLogPath, waiter: SearchPane.waitLoading });
```

Comment:
This login is unnecessary because you use only api methods

Reply:
fixed